models/tag/index.js

```diff
@@ -128,6 +128,25 @@ class Tag extends Model {
     return out;
   }
 
+  static async findTagsRelatedToTags(tagsArray) {
+    const query = `
+      FOR sortedTag in (
+        FOR t in tags FILTER t.tagname IN @givenTags
+          FOR v, e, p IN 2..2
+          ANY t
+          userTag
+          COLLECT foundTag = v AGGREGATE similarity = SUM(sqrt(p.edges[0].relevance * p.edges[1].relevance )) INTO u RETURN {created: foundTag.tagname, tagname: foundTag.tagname, relevance: similarity }
+      )
+      FILTER sortedTag.tagname NOT IN @givenTags
+      SORT sortedTag.relevance DESC, sortedTag.tagname ASC
+      RETURN sortedTag
+    `;
+    const givenTags = tagsArray;
+    const params = { givenTags };
```

Comment:
it's fine, but it's also possible to write `{ givenTags: tagsArray }` and remove the line above